src/events/Events.hpp

```diff
@@ -4,6 +4,7 @@
 // NOLINTNEXTLINE(readability-identifier-naming)
 namespace Events {
     // Window events
+    DYNLISTENFUNC(initializeWindow);
```

Comment:
this namespace is legacy and should no longer be used. I plan to get rid of it one day altogether. This fn should be a member of CWindow.